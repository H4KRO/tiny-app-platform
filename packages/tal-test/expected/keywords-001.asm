main()
  entry:
    ScopeEnter
    Local             name: letwww
    ScopeLeave        inBlock: false, count: 1
    Pop               inBlock: false
    ScopeEnter
    Local             name: setwww
    ScopeLeave        inBlock: false, count: 1
    Pop               inBlock: false
    ScopeEnter
    Local             name: ifwww
    ScopeLeave        inBlock: false, count: 1
    Pop               inBlock: false
    ScopeEnter
    Local             name: varwww
    ScopeLeave        inBlock: false, count: 1
