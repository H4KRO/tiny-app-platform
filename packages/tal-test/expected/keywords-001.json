[
  {
    "kind": "Block",
    "children": [
      {
        "kind": "Local",
        "name": "letwww"
      }
    ]
  },
  {
    "kind": "Block",
    "children": [
      {
        "kind": "Local",
        "name": "setwww"
      }
    ]
  },
  {
    "kind": "Block",
    "children": [
      {
        "kind": "Local",
        "name": "ifwww"
      }
    ]
  },
  {
    "kind": "Block",
    "children": [
      {
        "kind": "Local",
        "name": "varwww"
      }
    ]
  }
]